alica_engine/include/engine/BasicBehaviour.h

```diff
@@ -23,13 +23,20 @@ class TestContext;
 class Variable;
 class EntryPoint;
 
+struct BehaviourContext
+{
+    IAlicaWorldModel* worldModel;
+    const std::string name;
```

Comment:
the behaviourModel already has the name, so this is not required

Reply:
Earlier it was being given by the app behaviours so i left it there, just in case app wants to overwrite the default one.